docs/applications/container-repositories/02-github.md

```diff
@@ -0,0 +1,137 @@
+---
+id: applications-container-repositories-github
+slug: /applications/container-repositories/github
+title: GitHub Action
+sidebar_label: GitHub Action
+---
+
+# GitHub Action
+
+This guide will walk you through the process of setting up a GitHub Action to build, push, and deploy app changes to your chosen cloud container repository.
+
+:::note
+
+Before getting started, you'll need:
+- A Massdriver account 
+- A Massdriver [service account](/platform/service-accounts)
+
+:::
+
+## Publish your application
+
+Before you can set up a GitHub Action to deploy your application, first you'll need to publish it to Massdriver and create a package. You can do this by following the [Getting Started](/applications/getting-started) guide.
+
+After you publish your application, you'll need to create a package. You can do this by dragging your application out from the bundle bar in the Massdriver canvas. Fill in the values of your application and click **Deploy**.
+
+Copy and save the package slug by hovering over the name of your application on the canvas. This will be used in the next step.
+
+## Set secrets and vars
+
+Once you've published your application, you'll need to set the following secrets and vars in your GitHub repository:
+
+| Name | Description | Type | Notes |
+| --- | --- | --- | --- |
+| `MASSDRIVER_ORG_ID` | Your Massdriver organization ID | secret | Your Organization ID can be found by hovering over your org logo in the sidebar |
+| `MASSDRIVER_API_KEY` | Your Massdriver API key | secret | [Service Accounts](/platform/service-accounts) |
+| `AUTHENTICATION_ARTIFACT_ID` | The ID of the authentication artifact in Massdriver | secret | Select your cloud credential from the Artifacts page in Massdriver, then click **Copy Artifact ID** |
+| `NAMESPACE` | The namespace of your build | variable | If it does not exist paired with the `IMAGE_NAME`, then it will be created for you |
+| `IMAGE_NAME` | The image name of your build | variable | If it does not exist paired with the `NAMESPACE`, then it will be created for you |
+| `REGION` | The region where your cloud container repository is located | variable | Must be a valid cloud region. For example: `eastus` for Azure, or `us-west-1` for AWS. |
+
+## Workflow file
+
+:::note
+
+This GitHub Action will create a new cloud container repository with the following name format: `massdriver<region>`. The region will be the same as the region you set in the `REGION` variable. If the registry already exists, the action will use it.
```

Comment:
I thought it created `<namespace>:<image_name>`, not `massdriver<region>`

Reply:
I'll add clarification to this paragraph, but what I'm referring to is the cloud container repository instance (not the repo directory itself)

Reply:
important note here. On azure we create a new _registry_ massdriver<region><suffix>. Then we create a repository within that registry which is <namespace>/<image>. AWS already has the registry so we just create the repository. These details are probably best left out as they are implementation details the user shouldn't care about.